### PR TITLE
CI: Update postman collection on release

### PR DESCRIPTION
**Pull request by svix-frank**

### CODE REVIEW

File: tools/update_postman.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
+const axios = require('axios');
+const Converter = require('openapi-to-postmanv2');
+
+const POSTMAN_COLLECTION_ID = 'fdc9fd1f-d7bc-4f7e-b5ce-9189dbcbb81d';
+const POSTMAN_API_KEY = process.env.POSTMAN_API_KEY;
+const openapiUrl = 'https://api.svix.com/api/v1/openapi.json';
```
Comment (svix-frank) [inline tools/update_postman.js:7]: We want to use the prod json not the one stored in the repo so download it.

Comment (tasn) [reply tools/update_postman.js:8]: Why? We are doing it whenever the libs are released, let's just do whatever is stored locally.

File: tools/update_postman.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const update_collection = async (collection) => {
+  const res = await axios.put(
+    `https://api.getpostman.com/collections/${POSTMAN_COLLECTION_ID}`,
+    {
+      "collection": collection
+    },
+    {
+      headers: { "x-api-key": POSTMAN_API_KEY}
```
Comment (tasn) [inline tools/update_postman.js:26]:
```suggestion
      headers: { "x-api-key": POSTMAN_API_KEY }
```

File: tools/update_postman.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    );
+  console.log(res.body);
```
Comment (tasn) [inline tools/update_postman.js:29]: ?

**Review by tasn — DISMISSED** [review]

**Review by tasn — APPROVED** [review]

**Pull request merged**